conda/core/repodata.py

```diff
@@ -420,6 +420,12 @@ def fetch_repodata(url, schannel, priority,
     else:
         mod_etag_headers = read_mod_and_etag(cache_path)
 
+        if use_cache:
+            log.debug("Using cached repodata for %s at %s because use_cache=True",
+                      url, cache_path)
+            return read_local_repodata(cache_path, url, schannel, priority,
+                                       mod_etag_headers.get('_etag'), mod_etag_headers.get('_mod'))
+
```

Comment:
I considered that but decided against it so that it's easier to see from the log why the local cache is used. No strong opinion on it though, so I'm happy to change it if you prefer.